src/lib/core.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const EventEmitter = require('eventemitter2')
+const fiveBellsRouting = require('five-bells-routing')
+const Route = fiveBellsRouting.Route
+const RoutingTables = fiveBellsRouting.RoutingTables
+
+class Core extends EventEmitter {
+  constructor (routingTables) {
```

Comment:
Actually, this should be an object where `routingTables` is one field so we don't need a breaking API change to add new parameters